CGPs/cgp-0054.md

```diff
@@ -0,0 +1,183 @@
+
+**Prezenti** in Esperanto means to give, present, bring into the presence of. (Less frequent translations are: gift, offer, tender, feature, render).
+
+## 13. CGP Vote
+
+### 13.1 Smart Contract
+
+- 2-of-3 multi-signature smart contract that is controlled by the elected stewards. Our proposal is to have three stewards within our group.
+- The 2-of-3 CCF smart contract will be owned by the Governance smart contract so that CELO owners can claw back CELO via CGP, if required.
+
+### 13.2 Term
+
+- The following members would like to nominate themselves for the role of the stewards:
+    * Aaron Boyd
+    * Wade Abel
+    * Maya Richardson-Brown
+- The term we propose is until funds are spent with a goal of dispersing our budget to qualified projects within 12 months after the elections and the appropriate steward entity is created.
+
+### 13.3 Voting on the CCF2 proposal and the stewards
+
+We are asking the community to vote on the following (in one vote): 
+    1. On the revised proposal above
+    2. and for the stewards to manage this proposal
+
+This is due the to following reasons:
+- Prevent voter fatigue: by making them vote on two things. It’ll be easy to rally the Celo community behind one CGP.
+- Keep the Celo Community Fund flexible: We (Wade, Aaron and Maya) are applying as stewards and are requesting a certain amount to disperse. Another steward team could also make a similar proposal to use a part of the fund if needed.
+
+
+## Appendix:
+
+### Proposers:
+
+#### Aaron Boyd
+**Background:** Software engineer for nearly 20 years, worked the better part of 15 years in the financial services industry in Australia (banking, insurance, superannuation). Active in the blockchain space since 2013 and advocate for economic self-determination.
+
+**Current work:**  Founder and operator of VladiatorLabs.io, a protocol-aligned infrastructure company currently in the process of incorporating in Switzerland and expanding to other proof-of-stake networks, especially in the Cosmos ecosystem. Vladiator Labs is a founding project in the Staking Defense League, an infrastructure special interests group aimed at promoting decentralization in the staking space.
+Additionally has worked briefly as a contractor within cLabs on some minor projects. Involved in the Celo ecosystem since the incentivised testnet and have been supported by the Celo Foundation as an elected validator since genesis (over two different validator groups). 
+Freelance Solidity, blockchain, database and cloud engineer and recently has worked with RaveSpace on metaverse NFT gallery https://musee-dezentral.com.
+
+**Possible disclosures:** Vladiator Labs is a current CCF1 grant recipient for our block map and attestation tool https://vido.vladiatorlabs.io, a Foundation vote recipient, and a Foundation grant recipient for operating an ODIS signer as part of the phone number privacy module. 
```

Comment:
I should also note that I was the lead architect and engineer on a Flori-funded project [Humanity Cash](https://humanity.cash) and helped design and develop in a lesser capacity on the governance token design for [Impact Market](https://www.impactmarket.com/), another Flori venture. These ventures are already commercially funded and in production and will not require community grant funding.